dev-libs/pegtl/pegtl-3.2.0.ebuild

```diff
@@ -13,16 +13,10 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64 ~arm ~x86"
 IUSE="test"
-
 RESTRICT="!test? ( test )"
 
 S="${WORKDIR}/PEGTL-${PV}"
 
-PATCHES=(
-	"${FILESDIR}/${P}-cmake.patch"
-	"${FILESDIR}/${P}-gcc-10.patch" # bug 733678
-)
-
 src_configure() {
 	local mycmakeargs=(
 		-DPEGTL_INSTALL_CMAKE_DIR="$(get_libdir)/cmake/${PN}"
```

Comment:
if you're bored, try checkign whether upstream is willing to go for `GNUInstallDirs` (but I'm not asking you to do this)